tests/devapps/WAM/NetCoreWinFormsWam/Form1.cs

```diff
@@ -125,11 +125,11 @@ private IPublicClientApplication CreatePca(AuthMethod? authMethod)
                     break;
                 case AuthMethod.SystemBrowser:
                     builder.WithBroker(new BrokerOptions(BrokerOptions.OperatingSystems.None));
-                    builder = builder.WithBroker(false);
+                    builder = builder.WithBroker(false).WithSsoPolicy();
```

Comment:
suggest adding a check box for this in the form

Reply:
This can be done later.